docs/ai/development-workflow/protocols/94-batch-merge-protocol.md

```
@@ -0,0 +1,347 @@
+
+Any conflict that is not in the above categories, or a documentation file with overlapping changes.
+
+1. Display:
+
+   > **Non-trivial conflict detected in PR #N**
+   >
+   > Conflicting files:
+   > - `path/to/file.sh`
+   >
+   > Conflict excerpt:
+   > ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)